hardware/Netatmo.cpp

```diff
@@ -331,7 +331,7 @@ bool CNetatmo::RefreshToken(const bool bForce)
 			return true; //no need to refresh the token yet
 	}
 
-	Log (LOG_STATUS, "Requesting refreshed tokens");
+	Log (LOG_STATUS, "%s Requesting refreshed tokens", m_Name.c_str());
```

Comment:
This is not necessary, it will already add the hardware name Infront of each log message